### ISSUE TITLE
Second API call - similar (better) locations

### ISSUE THREAD

**Issue by bradleyjay** [bradleyjay/youre_better_than_this#15]
- [x] figure out what to go get - in category? Same star rating or above?
- [x] parse that from our current search result
- [x] package that up, new api call

**Comment by bradleyjay** [comment]
Current state: user-confirmation branch

tl;dr YES button should trigger the new business_search_suggestions() fcn, which we'll want to test next.
* Right now, search_results has been sent to location/verify.html to fill in the image, name, and so on
* How do we pass that search_result back from that form? Functions are ready to go (maybe) on the yelp_client.py (businesses_search_suggestions, get_better_url), but need to be passed search_results from the form (which is POSTing to /locations)
* Using placeholders on the new get_better_url(), not sure of the formatting that the API call for suggestions will require - will likely need helper functions to get the syntax right for Price, Categories, etc


Next steps:
- [ ] Get search_results from location/verify.html YES button press into the POST request that generates
- [ ] Ensure business_search_suggestions -> get_better_url uses proper syntax for the simple Yelp API call
- [ ] Build helper functions (if necessary) to get Price and Categories into Yelp API-friendly syntax
- [ ] (Later) Decide how to get location for the suggestions - ask the user where they want suggestions?
